kaggle/api/kaggle_api_extended.py

```diff
@@ -298,7 +306,7 @@ class KaggleApi(KaggleApi):
   config = os.path.join(config_dir, config_file)
   config_values = {}
   already_printed_version_warning = False
-  args = {}  # DEBUG Add --local to use localhost
+  args = {'--local'}  # DEBUG Add --local to use localhost

```

Comment:
It's a bit tedious to keep changing it during development. `DEBUG` is like `TODO` in that it won't be in the release. I'd prefer to leave it like this for now and change it prior to release.

Reply:
Can you clarify?  Is there some automated tool that strips this out?  I'm fine with it if it's automated, but not if it's a manual step that someone has to do prior to release, because e.g. someone else might do the release and not know that this needs to be done.

Reply:
Deleted.